protocol-configuration/permissions.ts

```diff
@@ -12,20 +12,20 @@ export const permissions = {
     'lusdPSMFeiSkimmer'
   ],
   GUARDIAN_ROLE: ['guardianMultisig', 'pcvGuardianNew', 'pcvSentinel'],
-  ORACLE_ADMIN_ROLE: ['optimisticTimelock', 'opsOptimisticTimelock', 'tribalCouncilTimelock'],
-  SWAP_ADMIN_ROLE: ['pcvEquityMinter', 'optimisticTimelock', 'tribalCouncilTimelock'],
+  ORACLE_ADMIN_ROLE: ['opsOptimisticTimelock', 'tribalCouncilTimelock'],
+  SWAP_ADMIN_ROLE: ['pcvEquityMinter', 'tribalCouncilTimelock'],
   BALANCER_MANAGER_ADMIN_ROLE: [],
   RATE_LIMITED_MINTER_ADMIN: [],
   PARAMETER_ADMIN: [],
   PSM_ADMIN_ROLE: ['tribalCouncilTimelock'],
-  TRIBAL_CHIEF_ADMIN_ROLE: ['optimisticTimelock', 'tribalCouncilTimelock'],
-  FUSE_ADMIN: ['optimisticTimelock', 'tribalCouncilTimelock'],
+  TRIBAL_CHIEF_ADMIN_ROLE: ['tribalCouncilTimelock'],
```

Comment:
you never delete this role in this PR but tom does in #922

Reply:
Not sure how I missed this - thanks